apps/sriov-kernel-nse/nse.yaml

```diff
@@ -0,0 +1,56 @@
+---
```

Comment:
@Bolodya1997 

Let's just reuse kernel-nse application and use kustomize feature `patchesStrategicMerge`. Take a look at LocalUsecase or RemoteUsecase

Reply:
Yep, removed both `sriov-kernel-nsc`, `sriov-kernel-nse`.